Model/Api/Product.php

```diff
@@ -379,14 +379,14 @@ protected function _buildProductData(
         $data["url"] = $product->getProductUrl();
         $data["image_url"] = '';
         if ($product->getImage() && $product->getImage()!='no_selection') {
-            $filePath = 'catalog/product'.$product->getImage();
+            $filePath = 'catalog/product/'.ltrim($product->getImage(), '/');
             $data["image_url"] = $this->_helper->getBaserUrl($magentoStoreId, \Magento\Framework\UrlInterface::URL_TYPE_MEDIA).$filePath;
         } elseif ($this->_parentImage) {
             $data['image_url'] = $this->_parentImage;
         } else {
             $parent = $this->_getParent($product->getId(), $magentoStoreId);
             if ($parent && $parent->getImage() && $parent->getImage()!='no_selection') {
-                $filePath = 'catalog/product'.$parent->getImage();
+                $filePath = 'catalog/product'.ltrim($parent->getImage(), '/');
```

Comment:
Hi @gonzaloebiz - you've missed the slash off the end of `catalog/product` it should be:

```
$filePath = 'catalog/product/'.ltrim($parent->getImage(), '/');
```